### PR TITLE
setup refactor

### PR DESCRIPTION
**Pull request by ashitadadlani**
Did some setup refactor changes

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -15,11 +15,9 @@
   ]
 )
 .config(
-  [          '$stateProvider', '$urlRouterProvider',  '$httpProvider',
-    function ($stateProvider,   $urlRouterProvider, $httpProvider) {
+  [          '$stateProvider', '$urlRouterProvider',
```
Comment (ruchirverma) [inline public/js/app.js:18]: Looks good

File: public/css/style-angular_material.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
 md-radio-button {
   display: inline;
+}
+ md-tab-item span {
```
Comment (wujingz) [inline public/css/style-angular_material.css:4]: Looks good

File: public/js/app.js (hunk 1 above)
Comment (ashitadadlani) [inline public/js/app.js:18]: Thanks pushing

**Comment by ashitadadlani** [comment]
:+1:  push it

**Pull request merged**